elyra/templates/components/canvas_palette_template.jinja2

```diff
@@ -10,8 +10,9 @@
     {% if component.runtime %}
       "runtime": "{{ component.runtime }}",
     {% endif %}
-    {% if component.extension %}
-      "extensions": ["{{ component.extension }}"],
+    {% if component.extensions %}
+      "extensions": ["{{ component.extensions }}"],
```

Comment:
Should probably double-check that because `component.extensions` is a list, this doesn't produce a nested list (e.g., `[['.ipynb']]`).  If so, it would be preferable if we could reduce this to `"{{ component.extensions }}"` rather than rebuild the list iteratively.  (Sorry, I'm not that familiar with jinja functionality.)

This seems like a good thing for a test assertion.

Reply:
I found a nice jinja keyword to make this work! Agreed in converting `extensions` to a list